components/match2/commons/match_group_util.lua

```diff
@@ -74,6 +74,7 @@ MatchGroupUtil.types.MatchCoordinates = TypeUtil.struct({
 MatchGroupUtil.types.MatchlistBracketData = TypeUtil.struct({
 	header = 'string?',
 	title = 'string?',
+	dateheader = 'boolean?',
 	type = TypeUtil.literal('matchlist'),
 })
```

Comment:
This describes the lua struct (result of `MatchGroupUtil.bracketDataFromRecord`) so it should be capitalized accordingly (`dateHeader` as you have it now)